templates/drupal8/files/.platform.template.yaml

```diff
@@ -23,7 +23,11 @@ info:
   # The human-readable name of the template.
   name: Drupal 8
   # Human-readable descriptive text for the template. Supports limited HTML.
-  description: Drupal is a highly-customizable free software PHP CMS that allows you to easily organize, manage and publish your content.
+  description: |
+    This template builds Drupal 8 using the [Drupal Composer project](https://github.com/drupal-composer/drupal-project)
```

Comment:
What do you think if we make this one shorter?
E.g.:
This template builds Drupal 8 using Composer project including  configuration to use Redis for caching [to enable](some link)

Reply:
Why does this need to be shorter?  The testing that Patrick did suggests people want more information here, not less.